operator/api/redpanda/v1alpha2/redpanda_clusterspec_types.go

```diff
@@ -1154,6 +1154,7 @@ type FsValidator struct {
 type Configurator struct {
 	ExtraVolumeMounts *string                      `json:"extraVolumeMounts,omitempty"`
 	Resources         *corev1.ResourceRequirements `json:"resources,omitempty"`
+	AdditionalCLIArgs []string                     `json:"additionalCLIArgs,omitempty"`
```

Comment:
As mentioned in slack we might want to perform this change in a separate commit.

Though now that I think about, it's probably fine to do this change at the same time. There will be a small period of time in the nightly builds where this field will be present but ignored by the helm chart. As long as everyone knows that it'll be fine. If we think there could be a misconception, two commits / PRs is likely worthwhile.

Reply:
I think @alenkacz is just about to kick off the Azure work. As long as we coordinate I think that should be the only user.